pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -145,7 +145,10 @@ func computeScheduleRecurrence(
 	}
 	expr, err := cronexpr.Parse(cron)
 	if err != nil {
-		return nil, err
+		return nil, errors.Newf(
```

Comment:
Will it be the same message for `RECURRING` and `FULL BACKUP`?

Reply:
well, this one will be set when we try to pick recurrence.  I added another file to this pr to make sure the error is the same(ish)

Reply:
Is the error that we get back here useful? If so, it might be worth wrapping?

Reply:
By here, I mean the error we get back from `cronexpr.Parse`.

Reply:
No, it's not useful at all.